src/Dfc.CourseDirectory.Web/ViewComponents/Apprenticeships/ChooseLocation/Default.cshtml

```diff
@@ -29,7 +29,29 @@
 
 <script>
     (function ($) {
-        $(function () {
+        $(function ()
```

Comment:
Does this need to be an AJAX request? Could maybe use formaction on the button and have it go direct to the action.

Reply:
Yes, with current flow, some how direct call on button was not navigating